source/css/style.css

```diff
@@ -1,20 +1,39 @@
-h4 small
-{
+.post-images {
+  margin-left: 20%;
+}
+.img-small {
+  height: 20px;
+  width: 20px;
+}
+.img-thumb {
+  height: 50px;
+  width: 50px;
+}
+.img-med{
+  height: 100px;
+  width: 100px;
+}
+.img-large {
+  height: 400px;
+  width: 400px;
+}
+.img-huge {
+  height: 800px;
+  width: 800px;
+}
+h4 small {
```

Comment:
Looking at this whole file, there is inconsistent indentation on the css.
https://github.com/waybetterdev/dev.waybetter.com/blob/WB-2/source/css/style.css#L30

Also please reread this:
https://github.com/waybetterdev/dev.waybetter.com/pull/9#discussion_r37329595 Unless @jasonrsavino told you to not add any blank lines between class definitions?